src/test/java/net/oauth/jsontoken/JsonTokenParserTest.java

```diff
@@ -33,6 +33,8 @@
 import java.security.SignatureException;
 import java.util.regex.Pattern;
 
+import static org.junit.Assert.assertThrows;
```

Comment:
nit: Place static imports above the non-static imports with one black line separating them (same for other files): https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Moved